Unison breaks bin/doctor.sh script

In the Docker Machine functionality check, we get this error:

```
ERROR: Volume conditioner-sync declared as external, but could not be found. Please create the volume manually using `docker volume create --name=projectname-sync` and try again.
```

Note that the doctor script has been very minimally maintained and is likely not much used. A secondary contribution after this problem is fixed and any other lurking bugs cleaned up is a re-evaluation in how this is implemented and possibly adding a project script to run bin/doctor.sh and rig doctor to .outrigger.yml

<details><summary>$> bin/doctor.sh</summary>

```
Docker Utilities
	[WARN] devtools command not found.
	[INFO] docker found
	[INFO] docker-compose found
	[INFO] docker-machine found
	[INFO] dev confirmed
Code Generated with Yo Outrigger Drupal
	[INFO] src found
	[INFO] package.json found
	[INFO] composer.json found
	[INFO] docker-compose.yml found
	[INFO] build.yml found
Project Requirements Met
	[ERROR] node_modules directory not found.
Code Generated by Grunt Drupal Tasks Build Process
	[ERROR] build directory not found.
	[ERROR] build/html directory not found.
	[ERROR] build/html/index.php file not found.
Docker-Machine Functionality
ERROR: Volume conditioner-sync declared as external, but could not be found. Please create the volume manually using `docker volume create --name=projectname-sync` and try again.
	[INFO] code volume-mount
Run "devtools doctor" to address Docker problems.
```
</details>

--- a/generators/environment/templates/bin/doctor.sh
+++ b/generators/environment/templates/bin/doctor.sh
@@ -94,9 +94,9 @@ assertNotEqual() {
   fi
 }
 
-if [ -z "$1" ] || [ "$1" == 'devtools' ]; then
+if [ -z "$1" ] || [ "$1" == 'outrigger' ]; then
   heading 'Docker Utilities'
-  assertCmd 'devtools' $WARNING
+  assertCmd 'rig' $WARNING
   assertCmd 'docker' $ERROR
   assertCmd 'docker-compose' $ERROR
   assertCmd 'docker-machine' $ERROR
@@ -124,14 +124,14 @@ if [ -z "$1" ] || [ "$1" == 'gdt' ]; then
   assertFile 'build/html/index.php' $ERROR
 fi
 
-if [ -z "$1" ] || [ "$1" == 'docker-machine' ]; then
-  heading 'Docker-Machine Functionality'
-  docker-compose -f build.yml run base ls /var/www/src > /dev/null
-  assertOk $? 'code volume-mount' $ERROR
+if [ -z "$1" ] || [ "$1" == 'unison' ]; then
+  heading 'Unison File Sync Functionality'
+  docker-compose -f build.yml run base ls /var/www/src > /dev/null 2>&1
+  assertOk $? 'unison file sync/volume bind mount' $ERROR
 fi
 
 if [ "$STATUS" -gt 0 ]; then
-  echo 'Run "devtools doctor" to address Docker problems.'
+  echo 'Run "rig doctor" to address general Docker problems.'
 fi
 
 exit $STATUS

